app/ota_client.py

```diff
@@ -372,6 +380,9 @@ def _update(self, version, url_base, cookies, event=None):
         self._update_phase = OtaClientUpdatePhase.METADATA
         url = f"{url_base}/"
         metadata = self._process_metadata(url, cookies)
+        total_regular_size = metadata.get_total_regular_size()
+        if total_regular_size:
+            self._statistics.set("total_file_size", total_regular_size)
```

Comment:
[NITS] 以下のように ` total_regular_file_size` にした方がよいと思いました。
```suggestion
        total_regular_file_size = metadata.get_total_regular_file_size()
        if total_regular_file_size:
            self._statistics.set("total_file_size", total_regular_file_size)
```